packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -763,10 +763,6 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
     return months - date2.getMonth() + date1.getMonth();
   }
 
-  _differenceInYears(date1, date2) {
-    return this._differenceInMonths(date1, date2) / 12;
-  }
```

Comment:
This method was added in https://github.com/vaadin/vaadin-date-picker/pull/49 and apparently not used since then.